fix for: linkage specification is incompatible with previous "_mm_malloc"
author: Kenneth Hoste (HPC-UGent)
--- pcl-pcl-1.8.1/common/include/pcl/pcl_macros.h.orig	2017-11-22 21:06:27.799438229 +0100
+++ pcl-pcl-1.8.1/common/include/pcl/pcl_macros.h	2017-11-22 21:07:16.950466785 +0100
@@ -380,7 +380,11 @@
 #endif
 
 #if defined (HAVE_MM_MALLOC)
-  #include <mm_malloc.h>
+  #if defined(__INTEL_COMPILER)
+    #include <malloc.h>
+  #else
+    #include <mm_malloc.h>
+  #endif // defined(__GNUC__)
 #endif
 
 inline void*
